src/normalizer.ts

```diff
@@ -222,6 +223,32 @@ rules.set('Transform const to singleton enum', schema => {
   }
 })
 
+rules.set('Transform nullable to null type', schema => {
+  if (schema.nullable !== true) {
+    return
+  }
+
+  delete schema.nullable
+
+  if (schema.const !== undefined) {
+    if (schema.const !== null) {
+      schema.enum = [schema.const, null]
+      delete schema.const
+    }
+  } else if (schema.enum) {
+    if (!schema.enum.includes(null)) {
+      schema.enum.push(null)
+      log('yellow', 'normalizer', 'enum should include null when schema is nullable', schema)
+    }
+  } else if (schema.type) {
+    schema.type = [...[schema.type].flatMap(value => value), 'null']
+  } else if (schema.anyOf) {
+    schema.anyOf.push(link({type: 'null'}, schema.anyOf))
+  } else if (schema.oneOf) {
+    schema.oneOf.push(link({type: 'null'}, schema.oneOf))
```

Comment:
@mdmower-csnw: Is this how AJV handles `nullable: true` for `anyOf` and `oneOf`? It doesn't say explicitly in the [docs](https://ajv.js.org/json-schema.html#nullable).